scheduler/src/cook/tools.clj

```diff
@@ -894,6 +894,15 @@
     (cond-> {:count 1 :cpus cpus :mem mem}
       gpus (assoc :gpus gpus))))
 
+(defn match-based-on-pool-name
```

Comment:
add a unit test?

Reply:
There is one via (get-default-container-for-pool)

Reply:
nitpick: I would make it separate still

Reply:
Ok. I added one. :)